web/src/api.util.test.js

```diff
@@ -12,4 +12,8 @@ it("should create the url to get a collection by its ID", () => {
   expect(apiUrlCollectionById(1)).toEqual(
     "http://api.lvh.me/collection/1/data"
   );
+
+  it("should create the url to get all devices", () => {
```

Comment:
We'll need this test to be on its own level, outside of the previous test.

 The cheatsheet to run tests are in the  `.github/pull_request_template##Checklist` file and section. To run a specific test, instead of all of them: `run test src/[sub-path-to-file]/[file-name.test.js]`